.github/workflows/test-cdn-jsdelivr.yml

```diff
@@ -18,32 +19,42 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - uses: pnpm/action-setup@v4
+      - name: Turborepo cache
+        uses: actions/cache@v4
+        with:
+          path: .turbo
+          key: ${{ runner.os }}-turbo-node-${{ matrix.node-version }}
+          restore-keys: |
+            ${{ runner.os }}-turbo-node-
+      - name: Use pnpm
+        uses: pnpm/action-setup@v4
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
           cache: 'pnpm'
       - name: Install dependencies
         run: pnpm install
-      - name: Turborepo cache
-        uses: dtinth/setup-github-actions-caching-for-turbo@v1
-      - name: Build Packages
+      - name: Build packages
         run: pnpm build:packages
-      - name: Start HTML Server
-        run: pnpm --filter @scalar-examples/cdn-api-reference dev &
-      - name: Get installed Playwright version
+      - name: Get Playwright version
         id: playwright-version
-        run: echo "PLAYWRIGHT_VERSION=$(node -e "console.log(require('./package.json').devDependencies['@playwright/test'])")" >> $GITHUB_ENV
-      # TODO doesn't work
-      - name: Cache playwright binaries
+        run: echo "PLAYWRIGHT_VERSION=$(npx playwright --version | cut -d' ' -f2)" >> $GITHUB_ENV
+      - name: Playwright binary cache
         uses: actions/cache@v4
         id: playwright-cache
         with:
-          path: '~/.cache/ms-playwright'
-          key: '${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}'
-          restore-keys: ${{ runner.os }}-playwright-
-      - name: Install Playwright browser binaries & OS dependencies
-        run: pnpm exec playwright install --with-deps
-      - name: Run e2e tests (jsdelivr)
+          path: |
+            ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-${{ steps.playwright-version.outputs.PLAYWRIGHT_VERSION }}
+          restore-keys: |
+            ${{ runner.os }}-playwright-
+      - name: Install Playwright browsers
+        run: npx playwright install
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+      - name: Install Playwright dependencies
+        run: npx playwright install-deps
```

Comment:
This looks good checking for the cache hit. Why do we use npx instead of pnpm for this install?

Reply:
Ah, just copied from somewhere. We can switch to pnpm for sure!

Reply:
It looks like this is how playwright recommends to install. LGTM